### PR TITLE
Update vote labels in snx index.json

### PR DESCRIPTION
**Pull request by andytcf**
Add better clarification on what each vote value is

**Add space for ...**  
  
Etherscan: https://etherscan.io/address/...  
CoinGecko: https://www.coingecko.com/en/coins/...  
Website: https://...  
Twitter: https://twitter.com/...

### CODE REVIEW

**Comment by andytcf** [comment]
Let me know if these values work and are not too long @bonustrack

File: spaces/synthetix/index.json [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
-  "symbol": "SNX",
+  "symbol": "Weighted Debt",
   "skin": "synthetix",
   "domain": "gov.synthetix.io",
   "strategies": [
     {
       "name": "synthetix",
       "params": {
-        "symbol": "SNX (WV)",
+        "symbol": "Quadratically Weighted Debt",
```
Comment (bonustrack) [inline spaces/synthetix/index.json:12]: This is a bit too long, it's breaking the layout, i would recommend to use abbreviation or acronym, ideally we should mention SNX too. Imagine this is more a symbol than a label. 
image

File: spaces/synthetix/index.json [hunk 2]
```diff
@@ -2,14 +2,14 @@
   "token": "0xC011a73ee8576Fb46F5E1c5751cA3B9Fe0af2a6F",
   "name": "Synthetix",
   "network": "1",
-  "symbol": "SNX",
+  "symbol": "Weighted Debt",
```
Comment (bonustrack) [inline spaces/synthetix/index.json:5]: Since there is only 1 strategy i think the global symbol should be the same as the strategy symbol

Comment (andytcf) [reply spaces/synthetix/index.json:5]: changed it both to Weighted Debt (nice and concise but more clear)

**Pull request merged**